packages/docs-site/src/content/docs/docs/plugins/bitwarden.mdx

```diff
@@ -64,34 +54,12 @@ const bitwardenPlugin = BitwardenSecretsManagerDmnoPlugin.injectInstance('bitwar
 If you are already using Bitwarden Secrets Manager, you likely already have existing [projects](https://bitwarden.com/help/projects/) that contain [secrets](https://bitwarden.com/help/secrets/). If so, now would be a good time to review how they are all organized. If not, you should create at least one project, as each secret can have a parent project it belongs to, and access can be granted to projects rather than managing each secret individually.
 
 :::tip[Use projects to segment access]
-You should use multiple projects to segment your secrets following the [Principle of Least Privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). This means at minimum, you should have one project for ultra-sensitive production secrets, and another for everything else. How much more you want to break things up is up to you and will depend on your security requirements.
-
-It is technically possible to create secrets without using projects and/or to assign permissions at the individual secret level, but we do not recommend it.
+You should use multiple projects to segment your secrets following the [Principle of Least Privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). See more in our [Secret segmentation](/docs/guides/secret-segmentation/) guide.
 :::
 
-## Setup Machine Account & Access Tokens
-
-Machine accounts can be granted access to projects, and each machine account can have multiple access tokens with optional expiration. How you want to manage this is up to you, but a sensible approach could be:
```

Comment:
this is all specific to bitwarden's auth model, so might be good to leave in? It is also still part of the setup process, so at least still need to mention setting them up, and can then defer to the segmentation guide more about the strategy?

Reply:
ok sorry, if you want add the bitwarden specific stuff that makes sense back in, go for it, or just revert the whole then on this file

Reply:
Leave it out for now and I’ll go back in and try to slim it down and link back to the guide